lib/jsdom/living/helpers/iterable-weak-set.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+function cleanup({ ref, set }) {
+  set.delete(ref);
+}
+
+/**
```

Comment:
No jsdoc please.

Reply:
Where was the decision to stop using **JSDoc** made?

Reply:
We've asked several times before; this is not news to you.

Reply:
I know, but I’m genuinely curious, especially because a good chunk of the **JSDoc** in this repository was contributed by _you_ according to `git blame`.